api/k3s_config.go

```diff
@@ -1,8 +1,9 @@
 package api
 
 type K3sServerConfig struct {
-	ClusterInit bool     `yaml:"cluster-init,omitempty"`
-	Token       string   `yaml:"token,omitempty"`
-	Server      string   `yaml:"server,omitempty"`
-	TLSSan      []string `yaml:"tls-san,omitempty"`
+	ClusterInit      bool     `yaml:"cluster-init,omitempty"`
+	Token            string   `yaml:"token,omitempty"`
+	Server           string   `yaml:"server,omitempty"`
+	TLSSan           []string `yaml:"tls-san,omitempty"`
+	DatatoreEndpoint string   `yaml:"datastore-endpoint,omitempty"`
```

Comment:
`main.go` makes no use of this new field. I don't see it being unmarshaled from yaml anywhere either. How is this supposed to be used?

Reply:
Yes, that's correct, it is not explicitly used. If k3s does not see that value in the configuration file, it will create an SQLite database in `/var/lib/rancher/k3s/server/db/state.db`.

We intended to use this in the 2 node project, for specifying the path for the kine database when using kubeadm.

If this helps, I can add explicit usage in main.go.

Reply:
ok as explained over a call, this is used on another repo (that consumes the struct from this one). No need to hardcode anything on main.go.

Reply:
Yes, that is supposed to be used by stylus. Not directly here.